packages/edit-site/src/style.scss

```diff
@@ -94,3 +93,7 @@ body.js.site-editor-php {
 }
 
 @include wordpress-admin-schemes();
+
+.components-popover.components-dropdown__content.dataforms-layouts-panel__field-dropdown {
```

Comment:
This style looks misplaced should we move it where the `dataforms-layouts-panel__field-dropdown` class is defined?

Reply:
That's actually better. Done at https://github.com/WordPress/gutenberg/pull/67024/commits/81726b35551fe25210b5c102d2a1ff82b7adb52f